cli/script/command-executor.ts

```diff
@@ -857,35 +857,53 @@ function getPackageMetricsString(obj: Package): string {
     return returnString;
 }
 
-function getReactNativeProjectAppVersion(platform: string, projectName: string): Promise<string> {
-    var missingPatchVersionRegex: RegExp = /^\d+\.\d+$/;
+function getReactNativeProjectAppVersion(platform: string, projectName: string, options: any): Promise<string> {
```

Comment:
I would prefer if you simply passed the `command` object or declared a new interface for `options` instead of declaring it as `any`